packages/apollo-server-types/src/index.ts

```diff
@@ -13,15 +13,48 @@ import {
 import { KeyValueCache } from 'apollo-server-caching';
 import { Trace } from 'apollo-engine-reporting-protobuf';
 
+export type BaseContext = Record<string, any>;
+export type DefaultContext = BaseContext;
```

Comment:
Not sure I understand why it's important to have DefaultContext and not just use BaseContext instead. I get that the names have different semantic implications though.